trunner/runners/IMXRT117xRunner.py

```diff
@@ -0,0 +1,47 @@
+#
+# Phoenix-RTOS test runner
+#
+# IMXRT117x runner
+#
+# Copyright 2021 Phoenix Systems
+# Authors: Damian Loewnau, Jakub Sarzyński
+#
+
+import time
+
+from .common import ARMV7M7Runner, GPIO
+
+
+class IMXRT117xRunner(ARMV7M7Runner):
+    """This class provides interface to run test case on IMXRT117x using RaspberryPi.
+       GPIO 4 must be connected to the pin 4 in P1 Header (using a resistor 3,3k).
+       GPIO 2 must be connected to an appropriate IN pin in relay module"""
+
+    SDP = 'plo-ram-armv7m7-imxrt117x.sdp'
+    IMAGE = 'phoenix-armv7m7-imxrt117x.disk'
+
+    def __init__(
+        self,
+        port,
+        phoenixd_port
+    ):
```

Comment:
Why not in one line.
```suggestion
    def __init__(self, port, phoenixd_port):
```